node/primitives/src/lib.rs

```diff
@@ -605,3 +605,11 @@ pub fn maybe_compress_pov(pov: PoV) -> PoV {
 	let pov = PoV { block_data: BlockData(raw) };
 	pov
 }
+
+/// How many votes we need to consider a candidate backed.
+///
+/// WARNING: This has to be kept in sync with the runtime check in the inclusion module and
```

Comment:
Since node side code now uses this function, it would be more accurate to just refer to runtime:

WARNING: This has to be kept in sync with the runtime check in the inclusion module.